espnet2/tasks/spk.py

```diff
@@ -186,6 +192,13 @@ def add_task_arguments(cls, parser: argparse.ArgumentParser):
             help="Directory of spk2utt file to be used in label mapping",
         )
 
+        group.add_argument(
+            "--spk_num",
+            type=int,
+            default=None,
+            help="specify the number of speakers during training",
```

Comment:
Does this mean the total number of speakers (classes)?

Reply:
yes, it's given automatically in spk.sh